src/main/java/seatsio/charts/CategoryUpdateParams.java

```diff
@@ -0,0 +1,28 @@
+package seatsio.charts;
+
+import com.google.gson.JsonObject;
+
+import static seatsio.json.JsonObjectBuilder.aJsonObject;
+
+public class CategoryUpdateParams {
+
+    private final String label;
+    private final String color;
+    private final Boolean accessible;
+
+    public CategoryUpdateParams(String label,
```

Comment:
For bonus points we could add `withXXX()` methods, to avoid having to pass in `null` to update just the color for example.

Reply:
There's a bit of mix'n'match going on with this approach, e.g. `CreateEventParams` vs `ChannelCreationParamsBuilder`. It might be a good idea to add some standardisation to this